cli/lsp/language_server.rs

```diff
@@ -413,6 +414,7 @@ impl Inner {
         LspError::internal_error()
       })?,
       documents: self.documents.clone(),
+      maybe_config_file: self.maybe_config_file.clone(),
```

Comment:
Is it expensive to clone?  If so maybe we should have an `Arc<ConfigFile>` instead?  We should use `Arc` more in passing around state like this I think anyways.  But also in light of the other comment, I think we should just have it be `lint_rules`.

Reply:
It might be, but I would need to do a lot of changes across the whole codebase to store `Option<Arc<ConfigFile>>` (including `module_graph.rs` and `program_state.rs`), see comment above.